ITE/6/README.md

```diff
@@ -0,0 +1,574 @@
+# ITE-6: Generalized link format
+
+Author: lodato@google.com
+
+Contributors/reviewers: dedic@google.com, nitinjain@google.com,
+patricklawson@google.com, tomhennen@google.com, wietse@google.com
+
+Date: September 2020
+
+# Abstract
+
+This ITE defines a new schema for in-toto link files, which are now generally
+called "attestations." An attestation has three distinct layers, mapping to the
+three distinct steps in verification. The innermost layer is user-defined to
+allow customers to define their own schemas; "link" is now one such user-defined
+schema.
+
+This specification is developed jointly with
+[Binary Authorization](https://cloud.google.com/binary-authorization), who will
+support the agreed-upon format once finalized. This way a single link file will
+be usable by either system.
+
+# Goals
+
+*   Standardize artifact metadata without being specific to the consumer (e.g.
+    in-toto or Binary Authorization). This way CI/CD pipelines, vulnerability
+    scanners, and other systems can generate a single set of attestations that
+    can be consumed by anyone.
+
+*   Make it possible to write policies (layouts) that take advantage of
+    structured information.
+
+# Specification
+
+## Introduction
+
+An **attestation** is the generalization of an in-toto link. It is a statement
+about an artifact, signed by an attester. Each attestation has a type indicating
```

Comment:
Does it need to be about one specific artifact? What about build steps that produce multiple ones? Does each of them end up having a separate attestation?

Reply:
We had thought there would be one attestation per artifact, but our work with Busytown made us realize we need to support multiple artifacts per-attestation.  This does make writing good policy somewhat harder.

Note that the example does have multiple artifacts specified.

Reply:
Clarified: "set of artifacts".